tests/test_utils/test_utils.py

```
@@ -0,0 +1,294 @@
+    try:
+        for apps in os.listdir(os.path.join(test_config["test_apps"], app)):
+            with open(os.path.join(test_config["test_apps"], app, apps)) as file:
+                metadata = json.load(file)
+                if image not in get_nested(
+                    metadata, ["DockerMatrix", "ImageNames"], []
+                ):
+                    return False
+        return True
+
+    except FileNotFoundError:
+        raise FileNotFoundError(f"App {app} does not exist.")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)